lab3/oolab/src/test/java/agh/ics/oop/AnimalTest.java

```diff
@@ -0,0 +1,123 @@
+package agh.ics.oop;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AnimalTest {
+
+    @Test
+    public void rightOrientationTest(){
+        //given
+        Animal zwierze = new Animal();
+
+        //when
+        MapDirection polnoc = MapDirection.NORTH;
+        MapDirection poludnie = MapDirection.SOUTH;
+        MapDirection wschod = MapDirection.EAST;
+        MapDirection zachod = MapDirection.WEST;
+
+        //test
+        assertEquals(polnoc, zwierze.getAnimalOrientation());
+
+        zwierze.move(MoveDirection.RIGHT);
+        assertEquals(wschod, zwierze.getAnimalOrientation());
+
+        zwierze.move(MoveDirection.RIGHT);
+        assertEquals(poludnie, zwierze.getAnimalOrientation());
+
+        zwierze.move(MoveDirection.RIGHT);
+        assertEquals(zachod, zwierze.getAnimalOrientation());
+
+
+        zwierze.move(MoveDirection.LEFT);
+        assertEquals(poludnie, zwierze.getAnimalOrientation());
+
+        zwierze.move(MoveDirection.LEFT);
+        assertEquals(wschod, zwierze.getAnimalOrientation());
+
+        zwierze.move(MoveDirection.LEFT);
+        assertEquals(polnoc, zwierze.getAnimalOrientation());
+
+    }
+
+
+
+    @Test
+    public void rightPositionsTest(){
+        //given
+        Animal zwierze = new Animal();
+
+        //when
```

Comment:
to nie do końca tak działa ;) `given` to wszystkie dane testowe (czyli również to co ma Pani w when i częściowo dalej), `when` to testowana metoda (czyli tutaj: wywołania `move()`), a `then` to asercje na tym, co nam wyszło.